### DISCUSSION TITLE
Yamaha Zone 2 Issue

### DISCUSSION THREAD

**Discussion by BobMac57** [music-assistant/support#2248] [Q&A] [answered]
I can access the Main Zone but, looking for independent access for Zone 2. I have a Yamaha RX-V6A, anyone able to use this zone and if so, how? FYI, everything works in MusicCast, HA and Alexa.

**Comment by OzGav** [comment, marked as answer]
Please see the fourth last item here https://music-assistant.io/faq/troubleshooting/

**Reply by BobMac57** [reply]
Thank you, I'll check it out.

**Reply by BobMac57** [reply]
This brings up another thought, what about adding MusicCast as a Provider?

**Reply by OzGav** [reply]
Vote here! https://github.com/orgs/music-assistant/discussions/409

**Comment by BobMac57** [comment]
> Vote here! https://github.com/orgs/music-assistant/discussions/409 I responded but not seeing a vote option.

**Reply by OzGav** [reply]
Just upvote the top post

**Reply by BobMac57** [reply]
Done but, not optimistic since the original request was almost 2 years ago... :(

**Reply by OzGav** [reply]
You never know it is somewhat up there in terms of use based on the HA analytics

![image](https://github.com/music-assistant/support/assets/19848947/847f07e9-3ac2-4490-b71c-f3e9aef1cd33)

All you need is for someone that has one and is a python dev and it will be done! :-)

**Reply by OzGav** [reply]
MusicCast has been added to 2.6.0b6
